From be35a8abe78749b278dfbc3fc71b48c88c25b337 Mon Sep 17 00:00:00 2001
From: Syrone Wong <wong.syrone@gmail.com>
Date: Thu, 9 May 2024 15:14:51 +0800
Subject: [PATCH 5/5] enable http log

---
 service/OneService.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/service/OneService.cpp b/service/OneService.cpp
index 88a516ebd..6fb426b08 100644
--- a/service/OneService.cpp
+++ b/service/OneService.cpp
@@ -221,6 +221,7 @@ bool bearerTokenValid(const std::string authHeader, const std::string &checkToke
 	return true;
 }
 
+#define ZT_DEBUG 1 /* enable http log */
 #if ZT_DEBUG==1
 std::string dump_headers(const httplib::Headers &headers) {
   std::string s;
-- 
2.45.0.windows.1

