From 1d260c338f76c620d35267d68b588ec9cd6d0ef6 Mon Sep 17 00:00:00 2001
From: Syrone Wong <wong.syrone@gmail.com>
Date: Thu, 9 May 2024 15:14:51 +0800
Subject: [PATCH 5/6] enable http log

---
 service/OneService.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/service/OneService.cpp b/service/OneService.cpp
index 594ff02d..468ea88b 100644
--- a/service/OneService.cpp
+++ b/service/OneService.cpp
@@ -226,6 +226,7 @@ bool bearerTokenValid(const std::string authHeader, const std::string &checkToke
 	return true;
 }
 
+#define ZT_DEBUG 1 /* enable http log */
 #if ZT_DEBUG==1
 std::string dump_headers(const httplib::Headers &headers) {
   std::string s;
-- 
2.47.0.windows.2

