From 97a178f3389488d6eb7de7ce6d7977bdc79b1b8d Mon Sep 17 00:00:00 2001
From: Syrone Wong <wong.syrone@gmail.com>
Date: Fri, 15 Nov 2024 11:26:06 +0800
Subject: [PATCH 6/6] MYSELF: fix missing WorkflowIdReusePolicy

---
 rustybits/smeeclient/src/lib.rs | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/rustybits/smeeclient/src/lib.rs b/rustybits/smeeclient/src/lib.rs
index d48876e2..081f917b 100644
--- a/rustybits/smeeclient/src/lib.rs
+++ b/rustybits/smeeclient/src/lib.rs
@@ -16,7 +16,7 @@ use serde::{Deserialize, Serialize};
 use std::str::FromStr;
 use std::time::Duration;
 use temporal_client::{Client, ClientOptionsBuilder, RetryClient, WorkflowClientTrait, WorkflowOptions};
-use temporal_sdk_core_protos::{coresdk::AsJsonPayloadExt, temporal::api::enums::v1::WorkflowIdReusePolicy};
+use temporal_sdk_core_protos::{coresdk::AsJsonPayloadExt, temporal::api::enums::v1::WorkflowIdReusePolicy, temporal::api::enums::v1::WorkflowIdConflictPolicy};
 use url::Url;
 use uuid::Uuid;
 
@@ -72,6 +72,7 @@ impl SmeeClient {
         println!("notifying network joined");
         let options = WorkflowOptions {
             id_reuse_policy: WorkflowIdReusePolicy::RejectDuplicate,
+            id_conflict_policy: WorkflowIdConflictPolicy::Fail,
             execution_timeout: None,
             run_timeout: None,
             task_timeout: None,
-- 
2.47.0.windows.2

