From dc58b8b00f2c291dad06307c03bbd1d8cbcd869c Mon Sep 17 00:00:00 2001
From: Syrone Wong <wong.syrone@gmail.com>
Date: Sat, 6 Aug 2022 21:58:28 +0800
Subject: [PATCH] add ext to include dir

---
 attic/world/build.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/attic/world/build.sh b/attic/world/build.sh
index d8800cd4c..a7bf28ce2 100755
--- a/attic/world/build.sh
+++ b/attic/world/build.sh
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-c++ -std=c++11 -I../.. -I.. -g -o mkworld ../../node/C25519.cpp ../../node/Salsa20.cpp ../../node/SHA512.cpp ../../node/Identity.cpp ../../node/Utils.cpp ../../node/InetAddress.cpp ../../osdep/OSUtils.cpp mkworld.cpp -lm
+c++ -std=c++11 -I../../ext -I../.. -I.. -g -o mkworld ../../node/C25519.cpp ../../node/Salsa20.cpp ../../node/SHA512.cpp ../../node/Identity.cpp ../../node/Utils.cpp ../../node/InetAddress.cpp ../../osdep/OSUtils.cpp mkworld.cpp -lm
